lib/store/app.ts

```diff
@@ -29,5 +29,11 @@ export const useStore = create<AppState>((set) => ({
     }));
   },
   theme: Theme.Light,
-  setTheme: (theme) => set({ theme }),
+  setTheme: (theme: Theme) => {
+    set(() => ({ theme }));
+    // update <html> class for Tailwind dark mode
+    if (typeof document !== 'undefined') {
+      document.documentElement.classList.toggle('dark', theme === 'dark');
+    }
+  },
```

Comment:
Mentioned above - but the theme needs more structure, so we're explicitly switching between color sets. This function has a code smell...